lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -229,13 +231,13 @@ defmodule ExUnit.Runner do
     end
   end
 
-  defp run_test(config, %{tags: tags} = test, context) do
+  defp run_test(%{tags: tags} = test, context, config) do
     EM.test_started(config.manager, test)
 
     test =
       if is_nil(test.state) do
-        capture_log? = Map.get(tags, :capture_log, config.capture_log)
-        run_test_with_capture_log(capture_log?, config, test, Map.merge(tags, context))
+        capture_log = Map.get(tags, :capture_log, config.capture_log)
```

Comment:
I have removed the `?` from this variables, as it is not only boolean values that will hold